addons/spreadsheet_dashboard/views/spreadsheet_dashboard_views.xml

```diff
@@ -5,66 +5,67 @@
         <field name="name">spreadsheet.dashboard.view.list</field>
         <field name="model">spreadsheet.dashboard</field>
         <field name="arch" type="xml">
-            <list create="false" editable="bottom">
+            <list create="false">
                 <field name="sequence" widget="handle" groups="base.group_system"/>
                 <field name="name"/>
                 <field name="group_ids" widget="many2many_tags" required="1"/>
                 <field name="company_ids" options="{'no_create': True}" widget="many2many_tags" groups="base.group_multi_company" placeholder="Visible to all"/>
+                <field name="dashboard_group_id" optional="hidden"/>
                 <field name="spreadsheet_binary_data" groups="base.group_no_one" widget="binary_spreadsheet" filename="spreadsheet_file_name" string="Data" />
```

Comment:
This field is not useful anymore in the list view since it's in the form view now (and you cannot do anything with it here in the list view)